codec/codec.go

```diff
@@ -69,3 +74,14 @@ type (
 		UnmarshalAminoJSON([]byte) error
 	}
 )
+
+type jsonAny struct {
+	JSONMarshaler
+	types.InterfaceRegistry
+}
+
+// NewJSONAnyMarshaler creates a JSONAnyMarshaler using JSONMarshaler
+// and InterfaceRegistry
+func NewJSONAnyMarshaler(jm JSONMarshaler, ir types.InterfaceRegistry) IfcJSONMarshaler {
```

Comment:
I believe all changes in this file are to be removed.

Reply:
Yes, these are old names before porting this to `master`. Git didn't flag it, but it's not used and we can remove it.